docs/README.md

```diff
@@ -49,17 +55,38 @@ There is also the option to add a new tag by typing it in the input field. Since
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image7.jpg" width="auto">
 
 ### 2. COMMENTING
-The second annotation type is a **comment**. Comments are text annotations, just like tags, with the essential difference that they are a free-text strings instead of predefined strings. If a user feels that the information contained in the tags is not adequate, they can add a custom text that conveys the information they want to add as an annotation.
+The second annotation type is a **comment**. Comments are text annotations, just like semantic tags, with the essential difference that they are a free-text strings instead of predefined strings. If a user feels that the information contained in the semantic tags is not adequate, they can add a custom text that conveys the information they want to add as an annotation.
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image8.jpg" width="auto">
 
-Comments can also be *Upvoted*, *Downvoted* or *Removed* by other users. 
+Comments can also be *Upvoted*, *Downvoted* or *Removed* by other users.
 
 ### 3. GEOTAGGING
-Lastly, **geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be *Upvoted*, *Downvoted* or *Removed* by other users.
+**Geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be *Upvoted*, *Downvoted* or *Removed* by other users.
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image9.jpg" width="auto">
 
+### 4. COLORTAGGING
+
+**Colortagging** is a special kind of tag that refers to color. A user can select the color from a predefined color pallette. Colortags can also be *Upvoted*, *Downvoted* or *Removed* by other users.
+
+<img src="_media/annotating-colortagging.png" width="auto">
+
+## Image Comparison
```

Comment:
Similarly with the previous comment, maybe something like "Image Comparison Campaign - Ranking a set of images"?